packages/javascript-sdk/package.json

```diff
@@ -13,6 +13,12 @@
   "homepage": "https://github.com/ForgeRock/forgerock-javascript-sdk#readme",
   "main": "./src/index.ts",
   "module": "./src/index.ts",
-  "exports": "./src/index.ts",
+  "exports": {
+    ".": {
+      "import": "./src/index.ts",
+      "require": "./src/index.ts"
+    },
+    "./src/oauth2-client/state-pkce": "./src/oauth2-client/state-pkce"
```

Comment:
had to add this for building. it isnt in the final output. but something to consider if we arent exporting this out of the root index

Reply:
Got it! I think keeping it mostly "internal" is actually good. I wouldn't want to publicly add more to the already large `index.ts` file.